Assets/Scripts/Systems/LevelGeneration/Valley.cs

```diff
@@ -0,0 +1,6 @@
+using UnityEngine;
+
+public class Valley
```

Comment:
Why does this class even exist when it's empty?

Reply:
Bedzie potrzebna w przyszłości